windows-iotcore/develop-your-app/MultiLang.md

```diff
@@ -26,12 +26,96 @@ See the below sample applications:
 
 ## Languages in OS
 
-Windows 10 IoTCore kits now include the language resources for the following languages
-
-* English : en-US, en-GB
-* French  : fr-FR, fr-CA
-* Spanish : es-ES, es-MX
-* Chinese : zh-CN
+Windows 10 IoTCore kits now include the language resources for the following languages:
+
+<table>
+<colgroup>
+<col width="50%" />
+<col width="50%" />
+</colgroup>
+<thead>
+<tr class="header">
+<th align="left">Language</th>
+<th align="left">Code</th>
+<th align="left">Region</th>
+</tr>
+</thead>
+<tbody>
+
+<tr class="odd">
+<td align="left"><p>Arabic</p></td>
+<td align="left"><p>ar-SA</p></td>
+<td align="left"><p>Asia</p></td>
+</tr>
+
+<tr class="odd">
+<td align="left"><p>German</p></td>
+<td align="left"><p>de-DE</p></td>
+<td align="left"><p>Europe</p></td>
+</tr>
+
+<tr class="odd">
+<td align="left"><p>Italian</p></td>
+<td align="left"><p>it-IT</p></td>
+<td align="left"><p>Europe</p></td>
+</tr>
+
+<tr class="odd">
+<td align="left"><p>Japanese</p></td>
+<td align="left"><p>ja-JP</p></td>
+<td align="left"><p>Asia</p></td>
+</tr>
+
+<tr class="odd">
+<td align="left"><p>Korean</p></td>
+<td align="left"><p>ko-KR</p></td>
+<td align="left"><p>Asia</p></td>
+</tr>
+
+<tr class="odd">
+<td align="left"><p>Dutch</p></td>
+<td align="left"><p>nl-NL</p></td>
+<td align="left"><p>Europe</p></td>
+</tr>
+
+<tr class="odd">
+<td align="left"><p>Polish</p></td>
+<td align="left"><p>pl-PL</p></td>
+<td align="left"><p>Europe</p></td>
+</tr>
+
+<tr class="odd">
+<td align="left"><p>Romanian</p></td>
+<td align="left"><p>ro-RO</p></td>
+<td align="left"><p>Europe</p></td>
+</tr>
+
+<tr class="odd">
+<td align="left"><p>Russian</p></td>
+<td align="left"><p>ro-RU</p></td>
+<td align="left"><p>Europe</p></td>
+</tr>
+
+<tr class="odd">
+<td align="left"><p>Greek</p></td>
+<td align="left"><p>el-GR</p></td>
+<td align="left"><p>Europe</p></td>
+</tr>
+
+<tr class="odd">
+<td align="left"><p>Portugese (Brazil)</p></td>
+<td align="left"><p>pt-BR</p></td>
+<td align="left"><p>South America/Europe</p></td>
+</tr>
+
+<tr class="odd">
+<td align="left"><p>Portguese (Portugal)</p></td>
+<td align="left"><p>pt-PT</p></td>
+<td align="left"><p>South America/Europe</p></td>
+</tr>
+
+</tbody>
+</table>
 
```

Comment:
Shouldn't these also be listed in the table?

Reply:
Yep! Good catch.